server/opts.go

```diff
@@ -432,6 +455,93 @@ func (o *Options) ProcessConfigFile(configFile string) error {
 	return nil
 }
 
+// VerifyConfig validates the configuration in case a server key
+// and signature file is present.
+func (o *Options) VerifyConfig() error {
+	if o.ConfigKey == "" || o.ConfigSigFile == "" {
+		return nil
+	}
+
+	key, err := ioutil.ReadFile(o.ConfigKey)
+	if err != nil {
+		return err
+	}
+
+	kp, err := nkeys.FromSeed(string(key))
+	if err != nil {
+		return err
+	}
+
+	sigfile, err := ioutil.ReadFile(o.ConfigSigFile)
+	if err != nil {
+		return err
+	}
+	configPath := filepath.Dir(o.ConfigFile)
+	verified := make(map[string]bool, 0)
+	for _, include := range o.includes {
+		verified[filepath.Join(configPath, include)] = false
+	}
+
+	// Verify entries in the sigfile.
+	check := func(signature []byte, fname string) error {
+		sig, err := base64.StdEncoding.DecodeString(string(signature))
```

Comment:
I think I use the non-padded one most of the time when dealing with signatures. Not sure if we want to be consistent here or not.